lib/fix.js

```diff
@@ -1,9 +1,18 @@
+function disableRules(ids = []) {
+	const rules = {};
+	for (const id of ids) {
+		rules[id] = 0;
+	}
+
+	return rules;
+}
+
 // (editor: Object) => function
 function fix(editor, lint) {
 	// (text: string) => Promise<void>
 	return async (text, exclude) => {
-		const fix = exclude ? report => !exclude.includes(report.ruleId) : true;
```

Comment:
Can't pass a function to a worker, so create a rules object with the disabled rules instead. Again, inspired by https://github.com/AtomLinter/linter-eslint/blob/8b724b2ed50ded3dd9a535283fd6ee17bc6dfcb6/src/main.js#L43-L54